core/src/main.cpp

```diff
@@ -152,7 +152,7 @@ constexpr auto default_jsonschema() -> std::string_view
 	// clang-format on
 	return R"({{
     "$schema": "https://json-schema.org/draft/2020-12/schema",
-    "$id": "https://schemas.irods.org/irods-s3-api/0.2.0/schema.json",
+    "$id": "https://schemas.irods.org/irods-s3-api/0.3.0/schema.json",
```

Comment:
can we get this from a variable?   so we don't have to touch it again?

Reply:
Would this be something in the CMakeLists.txt that carries over into main.cpp?

Reply:
Yes.

Can get that implemented for 0.4.0.

Reply:
okay, new issue please

Reply:
on it.

Reply:
https://github.com/irods/irods_client_s3_api/issues/128